src/metatrain/utils/architectures.py

```diff
@@ -172,8 +172,8 @@ def get_default_hypers(name: str) -> Dict:
     """Dictionary of the default architecture hyperparameters.
 
     :param: name of the architecture
-    :returns: default hyper paremeters of the architectures
+    :returns: default hyper parameters of the architectures
     """
     check_architecture_name(name)
     default_hypers = OmegaConf.load(get_architecture_path(name) / "default-hypers.yaml")
-    return OmegaConf.to_container(default_hypers)
+    return OmegaConf.to_container(default_hypers)["architecture"]
```

Comment:
I would maybe add a comment here that for a better UX we store the `default-hypers.yaml` of each architecture indented by one level (`architecture:`).